checkov/bitbucket/runner.py

```diff
@@ -26,7 +26,7 @@ def run(
         files: list[str] | None = None,
         runner_filter: RunnerFilter | None = None,
         collect_skip_comments: bool = True
-    ) -> Report:
+    ) -> Report | list[Report]:
```

Comment:
@gruebel Why?

Reply:
because the parent allows it to support runners with Image Referencer capabilities to return more than one report. it won't happen for Bitbucket, but for others, which use the `ObjectRunner` parent like GitLab CI, also GHA in the near future, etc.